### ISSUE TITLE
Creating/editing embedded dataobject

### ISSUE THREAD

**Issue by jonom** [sheadawson/silverstripe-shortcodable#47]
With this module I can embed an existing dataobject, and configure some additional parameters about how it should be embedded. So in the case of an image gallery for instance, I can embed one that already exists, but can't change what images are included in the gallery, or create a new one.

I'd like to allow this kind of direct editing for the linked dataobject and wondering if there's a recommended approach?

A simple solution might be to add a tab called e.g. 'Manage Image Galleries' that displays a gridfield for all the items of that class. I'm not sure how easy it would be to embed that editing process in a modal though?

Has anyone already solved this issue?

**Comment by sheadawson** [comment]
@jonom yeah this would make the module much more powerful. It's something I've thought about but not got much further than that. The modal would make things less straight forward but still very achievable I'm sure. I don't have time to do this myself... feel free to have a go and keep me posted if you feel inspired :)
